cibuildwheel/oci_container.py

```diff
@@ -133,14 +183,32 @@ def __enter__(self) -> Self:
         if detect_ci_provider() == CIProvider.travis_ci and platform.machine() == "ppc64le":
             network_args = ["--network=host"]
 
+        platform_args = [f"--platform={self.oci_platform.value}"]
+        if self.image.endswith(":cibw_local"):
```

Comment:
rather than forcing the tag to be `cibw_local, we could use a suffix or prefix that gets removed.
e.g. `cibw_local+myimage:latest` or `myimage:latest+cibw_local`

Reply:
This feels a bit arbitrary... I think I'd rather put an option on CIBW_CONTAINER_ENGINE. E.g. `CIBW_CONTAINER_ENGINE=docker; skip-pull=true`. But, if the reason for the `--pull=always` is a workaround for a bug with multiarch images, can we detect a multiarch image and add the flag only in that case? Or another, less targeted way would be to detect a local image and skip the pull for that?

Reply:
Could we perhaps do `docker image inspect {self.image} --format '{{.Architecture}}'` and only pull if the architecture is different from what we want?

Reply:
The usage of `cibw_local` tag is arbitrary. But the idea to use the already existing image options is not as it is directly related to just that, hence the proposal to prefix/suffix the image name.
I will investigate the `docker image inspect ...` option as it would provide a better UX.